sdk/python/requirements/py3.7-ci-requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
this seems wrong, should be 3.7?

Reply:
Right, I generated it wrong. Was wondering how to generate for each individual python version, then I realized I can just create another venv.